Bigtable/src/Table.php

```diff
@@ -80,6 +79,9 @@ class Table
      *     @type string $appProfileId This value specifies routing for
      *           replication. **Defaults to** the "default" application profile.
      *     @type array $headers Headers to be passed with each request.
+     *     @type int $retries Number of times to retry. **Defaults to** `3`.
+     *           This settings only applies to {@see mutateRows()}, {@see upsert()}
```

Comment:
These will need to include the FQNS for the doc site to render the links correctly (`Google\Cloud\Bigtable\Table::mutateRows()`)